waiter/src/waiter/core.clj

```diff
@@ -1178,16 +1147,30 @@
    :websocket-request-auth-cookie-attacher (pc/fnk [[:state passwords router-id]]
                                              (fn websocket-request-auth-cookie-attacher [request]
                                                (ws/inter-router-request-middleware router-id (first passwords) request)))
-   :websocket-request-acceptor (pc/fnk [[:state passwords server-name]
-                                        discover-service-parameters-fn]
+   :websocket-request-acceptor (pc/fnk [[:state server-name]
```

Comment:
Should this move to handler instead?

Reply:
I meant `:websocket-request-acceptor` moving to handler. 
We can also choose `websocket/make-websocket-request-acceptor` as the target for the function body to aggregate all the websocket methods in one file, `websocket.clj`.